charts/pytorchjob/templates/pytorchjob.yaml

```diff
@@ -21,7 +21,8 @@ metadata:
   {{- end }}
 spec:
 {{- if .Values.cleanPodPolicy }}
-  cleanPodPolicy: {{ .Values.cleanPodPolicy }}
```

Comment:
please also update chart version and changelog.

Reply:
Done